tests/serialization/test_dag_serialization.py

```diff
@@ -1796,3 +1796,38 @@ def x(arg1, arg2, arg3):
         "op_kwargs": {"arg1": [1, 2, {"a": "b"}]},
         "retry_delay": timedelta(seconds=30),
     }
+
+
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
Instead of filtering these, we should add `pytest.warns` to ensure the deprecation warning is emitted.

Reply:
What I wanted to do in this test is just to make sure it serialize properly the deprecation is a noise to this test.
It feels strange to add deprecation test here as this test is about serialization so I added this test in `test_dummy.py`
see https://github.com/apache/airflow/pull/22832/commits/ceead3ec82a5d685900d19dc018f4e1fab6723ac

Let me know if that is OK